### PR TITLE
Feat(dashboard description): allow multiple descriptions, enhance des…

### PR DESCRIPTION
**Pull request by PavelJiranek**
…cription configuration

@andriichumak as discussed on the plugins/embedding meeting, I'm sending a PR with the dashboard description plugin extensions.

### CODE REVIEW

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

File: dashboard_description_plugin/scripts/refresh-md.js [hunk 1]
```diff
@@ -9,7 +9,7 @@ dotenv.config({ path: path.resolve(__dirname, "../.env.secrets") });
 
 const workspace = process.env.WORKSPACE;
 const hostname = process.env.BACKEND_URL;
-const backend = "tiger";
+const backend = process.env.BACKEND_TYPE || 'bear';
```
Comment (andriichumak) [inline dashboard_description_plugin/scripts/refresh-md.js:12]: Could we please make "tiger" a default option?

File: dashboard_description_plugin/webpack.config.js [hunk 2]
```diff
@@ -39,7 +39,7 @@ module.exports = (_env, argv) => {
     const protocol = new URL(effectiveBackendUrl).protocol;
 
     const proxy = {
-        "/api": {
+        "/gdc": {
```
Comment (andriichumak) [inline dashboard_description_plugin/webpack.config.js:42]: This should depend on the backend type. In tiger it's `/api`.

**Review by andriichumak — CHANGES_REQUESTED** [review]
Looks great, thanks for preparing pull request. I have just two minor requests:
1. We should focus on Tiger first in the public examples. I made two comments on that.
2. Would it make sense to update the screenshot image that's used in the README of this example? I can see that the text description was updated with reacher content (i.e. images). Would be great to showcase this right away.

File: dashboard_description_plugin/scripts/refresh-md.js (hunk 1 above)
Comment (PavelJiranek) [reply dashboard_description_plugin/scripts/refresh-md.js:12]: Sure, I'll change it

Comment (PavelJiranek) [reply dashboard_description_plugin/scripts/refresh-md.js]: @andriichumak changed in https://github.com/gooddata/gooddata-plugin-examples/compare/973348080463185479653b6051b911c299044742..fbc943ef299d1d36c145d642099477acdd4447d6

File: dashboard_description_plugin/webpack.config.js (hunk 2 above)
Comment (PavelJiranek) [reply dashboard_description_plugin/webpack.config.js]: Thanks for pointing this out, I'll fix it in the internal plugins repo too. Here fixed in https://github.com/gooddata/gooddata-plugin-examples/compare/973348080463185479653b6051b911c299044742..fbc943ef299d1d36c145d642099477acdd4447d6

**Comment by PavelJiranek** [comment]
> Looks great, thanks for preparing pull request. I have just two minor requests:
> 
> 1. We should focus on Tiger first in the public examples. I made two comments on that.
> 2. Would it make sense to update the screenshot image that's used in the README of this example? I can see that the text description was updated with reacher content (i.e. images). Would be great to showcase this right away.

Both tiger-first related changes and an updated plugin image pushed in https://github.com/gooddata/gooddata-plugin-examples/compare/973348080463185479653b6051b911c299044742..fbc943ef299d1d36c145d642099477acdd4447d6

**Review by andriichumak — APPROVED** [review]

**Comment by yenkins** [comment]
### Sonar scan result


More detail, see in https://sonarqube-gate.intgdc.com/dashboard?id=gooddata-plugin-examples-gate-PR23

To scan for vulnerabilities in dependencies and run unit tests (to get coverage report in sonar) please comment your PR with 'extended check sonar'.

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-plugin-examples-fossa-scanning-zuul-el7/22/ : FAILURE in 3m 16s (non-voting)
- https://checklist.intgdc.com/job/compliance/job/gooddata-plugin-examples-sonarqube-scan-zuul-gate/22/ : SUCCESS in 45s
- https://checklist.intgdc.com/job/compliance/job/gooddata-plugin-examples-merge-compliance-check/23/ : SUCCESS in 19s

**Pull request merged**